src/gui/qgisinterface.h

```diff
@@ -397,6 +397,8 @@ class GUI_EXPORT QgisInterface : public QObject
     virtual QAction *actionDeletePart() = 0;
     //! Returns the native Vertex Tool action.
     virtual QAction *actionVertexTool() = 0;
+    //! Returns the native Vertex Tool for Active Layer action.
```

Comment:
Newly added API methods need to be tagged with the version they were added. So something like this is better:

    /**
     * Returns the native "Vertex Tool for Active Layer" action.
     * \see actionVertexTool()
     * \since QGIS 3.6
     */